pyproject.toml

```diff
@@ -17,11 +17,13 @@ packages = [
 include = [
     { path = "meson.build", format = "sdist" },
```

Comment:
Meson script won't be needed anymore.

Reply:
You should also remove `build` group from dependencies

Reply:
Done!